README.md

```diff
@@ -115,7 +121,8 @@ These two operations are already defined in key mappings:
 ### Routers
 
 - `require('gitlinker.routers').blob`: generate the `/blob` url, by default `link` API will use this router.
-- `require('gitlinker.routers').blame` (todo): generate the `/blame` url.
+- `require('gitlinker.routers').blame`: generate the `/blame` url.
+- `require('gitlinker.routers').src`: generate the `/src` url (for [BitBucket.org](https://bitbucket.org/)).
```

Comment:
I think `/src` should not be considered as a separate router. We should have two main routers `blob` and `blame`.

In `blob` router we should decide the url keyword like `blob`, `src` or `tree` based on host pattern. Similar to [vim-gh-line.vim](https://github.com/ruanyl/vim-gh-line/blob/master/plugin/vim-gh-line.vim#L221)

Same behaviour can be achieved for the `blame` router. Here we can decide the keyword `blame` or `annotate` as per host type.

I am suggesting this because current implementation will enforce user to created separate binding for each host type with different router config. In my opinion user should be creating the mapping based on action and router irrespective of the git host.

Reply:
Would recommend to rename the `blob` router to `browse`, that would make more sense if backward compatibility is not a concern.

Reply:
Hi, @G0V1NDS, your point makes sense.

for now I am using the `router_binding` option to auto bind different routers to different host.

but just like you said, the `blame` is still need to be manually configured for different hosts.

I will refactor this part.

again thanks for your contribution!

Reply:
hi @G0V1NDS , I submit PR: https://github.com/linrongbin16/gitlinker.nvim/pull/118, would you please help review?

Reply:
Sure, will check in few hours.

Reply:
Added comment on router_binding implementation, see if that makes sense.